android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -443,6 +476,21 @@ fun getPromotionalOffer() : ErrorContainer {
 
 // region private functions
 
+internal class InvalidProrationMode(): Exception()
```

Comment:
Maybe rename to `InvalidProrationModeException`? Not sure if that's the Java dev in me 😅